lib/linguist/languages.yml

```diff
@@ -6019,6 +6019,13 @@ fish:
   tm_scope: source.fish
   ace_mode: text
   language_id: 415
+mIRC Script:
+  type: programming
+  extensions:
+  - ".mrc"
```

Comment:
[Thanks to @Alhadis](https://github.com/Alhadis/Harvester/issues/13#issuecomment-535805452), I counted 234 repositories by 220 users with `.mrc` files matching the search query provided in this pull request. There are a lot more files on GitHub than the search query finds. A lot of them seem to be binary files; I couldn't find another language using `.mrc`. I think we're fine here.